.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "julia.environmentPath": "/home/b4c/Documents/code/KolmogorovArnold.jl/test"
+}
```

Comment:
This file does not belong here. You should likely ignore `.vscode/` in the global `.gitignore` on your machine.

Reply:
do a git rm on this file